src/main/java/hw4/testdata/MetalColorPageTestData.java

```diff
@@ -0,0 +1,101 @@
+package hw4.testdata;
+
+import java.util.List;
+
+public class MetalColorPageTestData {
+
+    private List<String> summaryList;
+    private List<String> elementsList;
+    private String color;
+    private String metal;
+    private List<String> vegetablesList;
+
+    public MetalColorPageTestData(List<String> summaryList, List<String> elementsList, String color, String metal, List<String> vegetablesList) {
```

Comment:
Why this constructor public? As I could see that you use `Builder` design pattern.

Reply:
fixed, changed to private